eos/utils/kinematic.cc

```diff
@@ -200,9 +268,10 @@ namespace eos
         return _imp->variables.end();
     }
 
-    KinematicVariable::KinematicVariable(const std::shared_ptr<Implementation<Kinematics>> & imp, unsigned index) :
+    KinematicVariable::KinematicVariable(const std::shared_ptr<Implementation<Kinematics>> & imp, unsigned index, bool is_alias) :
         _imp(imp),
-        _index(index)
+        _index(index),
+        _is_alias(is_alias)
```

Comment:
I don't think ``_is_alias`` is required anymore now. However, I'm happy to leave it in, it might be useful later and does it does not put pressure on memory consumption.